plugins/autopeering/parameters.go

```diff
@@ -5,13 +5,16 @@ import (
 )
 
 const (
-	// CfgEntryNodes defines the config flag of the entry nodes.
-	CfgEntryNodes = "autopeering.entryNodes"
-	// CfgNetworkVersion defines the config flag of the network version.
-	CfgNetworkVersion = "autopeering.networkVersion"
+	// CfgMana defines the config flag of mana in the autopeering.
+	CfgMana = "autopeering.mana"
+	// CfgR defines the config flag of R.
+	CfgR = "autopeering.R"
+	// CfgRo defines the config flag of Ro.
+	CfgRo = "autopeeringRo"
 )
 
 func init() {
-	flag.StringSlice(CfgEntryNodes, []string{"2PV5487xMw5rasGBXXWeqSi4hLz7r19YBt8Y1TGAsQbj@ressims.iota.cafe:15626", "5EDH4uY78EA6wrBkHHAVBWBMDt7EcksRq6pjzipoW15B@entryshimmer.tanglebay.com:14646"}, "list of trusted entry nodes for auto peering")
-	flag.Int(CfgNetworkVersion, 21, "autopeering network version")
+	flag.Bool(CfgMana, true, "Enable/disable mana in the autopeering")
```

Comment:
@capossele would you like to also refactor parameters as messagelayer?
or just put it to the TODO list for now :stuck_out_tongue: